src/main/java/com/microsoftopentechnologies/windowsazurestorage/AzureStorageBuilder.java

```diff
@@ -49,18 +63,29 @@
 	private String downloadDirLoc;
 	private boolean flattenDirectories;
 	private boolean includeArchiveZips;
+	private BuildSelector buildSelector;
+	private String projectName;
 
 	@DataBoundConstructor
-	public AzureStorageBuilder(String storageAccName, String containerName,
+	public AzureStorageBuilder(String storageAccName, String containerName, BuildSelector buildSelector,
```

Comment:
Does CopyArtifact become a necessary dependency of Azure Storage plugin after this change?